extensions/test-backend-controller/src/main/java/de/sovity/edc/extension/testbackendcontroller/TestBackendController.java

```diff
@@ -45,4 +47,11 @@ public void setDataSinkValue(String incomingData) {
     public String echoForDataSource(@QueryParam("data") String message) {
         return message;
     }
+
+    @GET
+    @Path("/data-source-params")
```

Comment:
data source was the name of the endpoint, we could do it like that `/data-source/echo-query-params`